.buildkite/pipeline.deploy-prod.yml

```diff
@@ -4,7 +4,7 @@ steps:
     concurrency_group: "experience-deploy"  
     plugins:
       - docker#v3.5.0:
-          image: wellcome/weco-deploy:5.6.16
+          image: 760097843905.dkr.ecr.eu-west-1.amazonaws.com/wellcome/weco-deploy:5.6.16
```

Comment:
You can use `5.6` now to point at the most recent release in that range.

Reply:
Thanks to @jamieparkinson for that change. ^

Reply:
Will do didn't want to actually change more to minimise places for debugging if it borked